airflow-core/docs/best-practices.rst

```diff
@@ -650,6 +650,127 @@ want to optimize your Dags there are the following actions you can take:
 Testing a Dag
 ^^^^^^^^^^^^^
 
+
+Testing Operators with pytest
+-----------------------------
+
+The ``dag.test()`` example shown in older documentation works only when executed
```

Comment:
I think we can remove this paragraph - there is no point to refer to "past" - the important is "current" - this is something we tend to forget that if someone is used to old ways of doing things, they got it from somewhere (old docs, internet) - but what our documentation shoudl reflect is what is "now" best practice.